template/lib/default.nix

```diff
@@ -0,0 +1,5 @@
+{ lib }:
+
+{
+  overlayToPkgs = import ./overlays-to-pkgs.nix { inherit lib; };
```

Comment:
Maybe move this function to soxin.

Reply:
And acknowledge it to https://github.com/nrdxp/nixflk. Let's also add an "acknowledgments" section in the readme.